src/application/components/icons/Settings.tsx

```diff
@@ -0,0 +1,32 @@
+import { Icon, IconProps } from "../Icon";
+
+interface SettingsIconProps {
+  className?: string;
+  size?: IconProps["size"];
+}
+
+export function SettingsIcon({ className, size }: SettingsIconProps) {
+  return (
+    <Icon viewBox="0 0 24 24" size={size} className={className}>
```

Comment:
I got SVGR working, so it might be smarter to move these to `.svg` files.

Reply:
Looks like we've got the new icons coming soon in a new version of the brand package. I'll hold off updating these until that new version is released.